pkg/cmd/pipeline/translate.go

```diff
@@ -278,6 +275,7 @@ func updateCmap(cmap *apiv1.ConfigMap, data *CfgData) error {
 	cmap.Data[yamlField] = base64.StdEncoding.EncodeToString(data.Manifest)
 	cmap.Data[iconField] = data.Icon
 	cmap.Data[actionNameField] = actionName
+	cmap.Data[variablesField] = translateVariables(data.Variables)
```

Comment:
Post-merge review:
```suggestion
vars := translateVariables(data.Variables)
if vars != "" {
  cmap.Data[variablesField] = translateVariables(data.Variables)
}
```

Reply:
We depend on checking if the [variables field is set](https://github.com/okteto/okteto/blob/master/cmd/up/exec.go#L275). That's why e2e tests are failing